pkg/daemons/agent/agent_windows_test.go

```diff
@@ -1,47 +0,0 @@
-//go:build windows
-// +build windows
-
-package agent
-
-import (
-	"testing"
-
-	"github.com/k3s-io/k3s/pkg/daemons/config"
-)
-
-func TestCheckRuntimeEndpoint(t *testing.T) {
```

Comment:
I'm assuming that this use case is now covered by another test?

Reply:
no, this test was broken by design and never ran. It is literally just defining an empty map and then checking for a key in that map without running any code that would modify it:
```golang
			argsMap := map[string]string{}
			if argsMap["container-runtime-endpoint"] != tt.want {
```